controller/api.controller.js

```diff
@@ -0,0 +1,8 @@
+const endpointsJson = require("../endpoints.json");
+
+function getApi(req, res) {
+  console.log(endpointsJson);
```

Comment:
Please remove console logs before pushing work. Great to see you're using them during development but just remember to clean them up.